product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx

```diff
@@ -5,13 +5,16 @@ title: "Certified and supported product versions"
 You can use the following database product versions with Replication Server:
 
 -   PostgreSQL versions 11, 12, 13, 14, and 15
--   Advanced Server versions 11, 12, 13, 14, and 15
+-   EDB Postgres Advanced Server versions 11, 12, 13, 14, and 15
 -   Oracle 11g Release 2 version 11.2.0.2.0 is explicitly certified. Newer minor versions in the 11.2 line are supported as well.
 -   Oracle 12c version 12.1.0.2.0 is explicitly certified. Newer minor versions in the 12.1 line are supported as well.
 -   Oracle 18c version 18.1.0.2.0 is explicitly certified. Newer minor versions in the 18.1 line are supported as well.
 -   Oracle 19c version 19.1.0.2.0 is explicitly certified. Newer minor versions in the 19.1 line are supported as well.
 -   SQL Server 2014 version 12.0.5000.0 is explicitly certified. Newer minor versions in the 12.0 line are supported as well.
 
+!!!Note
+   See BigAnimal's [database version policy](/biganimal/latest/overview/05_database_version_policy/) for the versions of PostgreSQL and EDB Postgres Advanced Server available using [BigAnimal](/biganimal/latest/) (EDB’s managed database cloud service). 
```

Comment:
I think this note needs to be more explicit that Replication Server supports the use of BigAnimal PostgreSQL and EPAS clusters.  Perhaps lead with a sentence like the following:
> All the versions of PostgreSQL and Advanced Server available via BigAnimal (EDB’s managed database cloud service) also supported.

Reply:
After some addition thought on this, I think we need to be even more explicit with the note.  Here's what I now propose:

> All PostgreSQL and Advanced Server versions available as BigAnimal Single Node and Primary/Standby High Availabilty cluster types are also supported for SMR configurations.  Consult the [BigAnimal](https://www.enterprisedb.com/docs/biganimal/release/) (EDB’s managed database cloud service) documentation for more information about BigAnimal’s [supported cluster types](https://www.enterprisedb.com/docs/biganimal/release/overview/02_high_availability/) and [database version policy](https://www.enterprisedb.com/docs/biganimal/release/overview/05_database_version_policy/) for the versions of PostgreSQL and EDB Postgres Advanced Server available in BigAnimal.

Reply:
done